src/OmniSharp/Startup.cs

```diff
@@ -109,8 +108,8 @@ public static OmnisharpWorkspace CreateWorkspace()
         {
             var assemblies = MefHostServices.DefaultAssemblies;
 #if DNX451
-            assemblies = assemblies.AddRange(RoslynCodeActionProvider.MefAssemblies);
-            assemblies = assemblies.AddRange(NRefactoryCodeActionProvider.MefAssemblies);
+//             assemblies = assemblies.AddRange(RoslynCodeActionProvider.MefAssemblies);
+//             assemblies = assemblies.AddRange(NRefactoryCodeActionProvider.MefAssemblies);
```

Comment:
Had to comment the refactoring code out. Need to wait until we can get a signed compatible version of the Roslyn features dlls for -rc3